cmd/shield/main.go

```diff
@@ -52,10 +53,14 @@ func main() {
 		After:     getopt.StringLong("after", 'A', "", "Only show archives that were taken after the given date, in YYYYMMDD format."),
 		Before:    getopt.StringLong("before", 'B', "", "Only show archives that were taken before the given date, in YYYYMMDD format."),
 		To:        getopt.StringLong("to", 0, "", "Restore the archive in question to a different target, specified by UUID"),
-		Limit:     getopt.StringLong("limit", 0, "", "Display only the X most recent tasks or archives"),
+		Limit:     getopt.StringLong("limit", 0, "", "Display only the X most recent tasks, archives, or users"),
 
 		Full: getopt.BoolLong("full", 0, "Show all backend information when listing backends"),
 
+		Backend: getopt.StringLong("backend", 'b', "", "Only show users with the specified backend."),
```

Comment:
Where is this option used?  Looks to me like `shield users` only shows local users...

Reply:
Yeah thats true, I had functionality to list users from all backends but hardcoded it to local as the request was for LUM, (not sure if we want to include that via a backend flag to show other than local)

Reply:
You also (shouldnt/wouldnt) be able to edit/delete/etc non local backend users as that is all handled via the mapping.